README.md

```diff
@@ -151,6 +146,11 @@ Optionally, you may pretend to be an AMP Cache:
   3. Ensure the packager is still running; it's needed to serve the certificate.
   4. Visit `http://localhost:8000/` in the experimental Chromium.
 
+### How will these web packages be discovered?
+
+The details of that are still being worked out. We have several alternatives in
+mind and want to come up with an answer that's best for the web.
```

Comment:
Maybe add a comment along the lines of: "If you have constraints or suggestions, please reach out to..."